src/App.svelte

```diff
@@ -367,6 +367,11 @@
             >Terms</a
           >
         </li>
+        <li class="link-item">
+          <a class="link-item-anchor" href="/license_report.html"
```

Comment:
We need to avoid using "/" because our site is in a subdir.
This links to https://google-research.github.io/license_report.html, the correct link is https://google-research.github.io/vaccination-search-insights/license_report.html